trace-gradle-plugin/src/functionalTest/java/io/bitrise/trace/plugin/util/FunctionalTestWriter.java

```diff
@@ -17,7 +18,7 @@
     private static final char[] indentation = new char[]{' ', ' ', ' '};
     private static final char[] escapeChar = new char[]{27};
     private final char[] coloringCode;
-    private final int extraLength;
+    private final char[] coloringEnd = concatEscapeChar("[39;49m".toCharArray());
```

Comment:
what does `[39;49m` mean here?

Reply:
That is the required ANSI code to end the coloring.